src/test/java/org/folio/linked/data/util/JsonUtilsTest.java

```diff
@@ -19,6 +19,6 @@ void merge_shouldReturnModifiedIncoming() throws JsonProcessingException {
     var incoming = loadResourceAsString("samples/json_merge/incoming.jsonl");
     var merged = loadResourceAsString("samples/json_merge/merged.jsonl");
     var result = JsonUtils.merge(MAPPER.readTree(existing), MAPPER.readTree(incoming));
-    assertThat(result.equals(MAPPER.readTree(merged))).isTrue();
+    assertThat(result).isEqualTo(MAPPER.readTree(merged));
```

Comment:
With this change, we will be able to see the actual JSON and expected JSON in terminal if the test fail.

Reply:
this equals is overrided version of JsonNode
<img width="400" alt="image" src="https://github.com/user-attachments/assets/3540b1b1-dbaa-46b5-b4f1-d456fb4de14f">